tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
 
     public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
 
+    public static Map<String,Object> BUILD_PROPERTIES = null;
```

Comment:
Also lets not use upper case for BUILD_PROPERTIES, because this makes it look like a constant when it certainly changes.